backstage/packages/backend/src/plugins/scaffolder/templates/rad-lab-data-science-create.test.ts

```diff
@@ -92,29 +92,43 @@ describe('rad-lab-data-science-create: fetch:template', () => {
       });
 
       expect(mockDir.content({ path: 'workspace' })).toMatchInlineSnapshot(`
-      {
-        "claim.yaml": "---
-      apiVersion: data-science-portal.phac-aspc.gc.ca/v1alpha1
-      kind: RadLabDataScienceClaim
-      metadata:
-        name: <project-name>-<uuid>
-      spec:
-        rootFolderId: '<root-folder-id>'
-        projectName: <project-name>
-        projectId: <project-id>
-        projectEditors:
-          - user:jane.doe@gcp.hc-sc.gc.ca
-          - user:john.doe@gcp.hc-sc.gc.ca
-        projectViewers:
-          - user:samantha.jones@gcp.hc-sc.gc.ca
-          - user:alex.mcdonald@gcp.hc-sc.gc.ca
-          - user:john.campbell@gcp.hc-sc.gc.ca
-        labels:
-          cost-centre: '<cost-centre>'
-          vanity-name: '<project-name>'
-      ",
-      }
-    `);
+        {
+          "catalog-info.yaml": "---
+        apiVersion: backstage.io/v1alpha1
+        kind: Component
+        metadata:
+          name: <project-id>
+          title: <project-name>
+          annotations:
+            backstage.io/source-location: https://github.com/PHACDevHub/sci-portal/
```

Comment:
We're missing the path to the folder here. Have a look at the changes I made to the project-create.test.ts last night. It'll be like this above:

```diff
           catalogEntityOwner: 'user:default/jane.doe',
+          sourceLocation: 'DMIA-PHAC/SciencePlatform/<project-id>/',
         },
```

Then

```diff
-          backstage.io/source-location: https://github.com/PHACDevHub/sci-portal/
+          backstage.io/source-location: https://github.com/PHACDevHub/sci-portal/DMIA-PHAC/SciencePlatform/<project-id>/

```

Reply:
It looks like we need it below too.